shared/constants/wallets.js

```diff
@@ -163,6 +164,9 @@ const getFederatedAddress = (state: TypedState, accountID?: Types.AccountID) =>
   return username && account.isDefault ? `${username}*keybase.io` : ''
 }
 
+const getSecretKey = (state: TypedState, accountID: Types.AccountID) =>
```

Comment:
likely better to return a string always so you don't have to check and call stringValue() later